recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ocp" %}
+{% set version = "7.5.2beta" %}
+{% set release_version = "7.5.2-beta" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # currently only used for the license file
+  - url: https://github.com/CadQuery/OCP/archive/refs/tags/{{ release_version }}.tar.gz
+    folder: ocp
+
+  # temporarily use personal fork until CadQuery/OCP includes source releases
+  - url: "https://github.com/whophil/OCP/releases/download/{{ release_version }}/OCP_src_windows-2019.zip"  # [win]
+  - url: "https://github.com/whophil/OCP/releases/download/{{ release_version }}/OCP_src_macOS-10.15.zip"   # [osx]
+  - url: "https://github.com/whophil/OCP/releases/download/{{ release_version }}/OCP_src_Ubuntu-16.04.zip"  # [linux]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('libxi-devel') }}       # [linux]
+    - {{ cdt('libxext-devel') }}     # [linux]
+  host:
+    - python
+    - pybind11
+    - pybind11-abi
+    - occt 7.5.2
+    - vtk
+  run:
+    - python
+    - {{ pin_compatible('occt', max_pin='x.x.x') }}
+    - vtk
+
+test:
+  imports:
+    - OCP
+
+about:
+  home: https://github.com/CadQuery/OCP/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: ocp/LICENSE
+  summary: 'Python wrapper for OCCT using auto-generated pybind11 bindings.'
+  doc_url: https://github.com/CadQuery/OCP/blob/master/README.md
+  dev_url: https://github.com/CadQuery/OCP/
+
+extra:
+  recipe-maintainers:
+    - whophil
+    - adam-urbanczyk
```

Comment:
would need a comment from the non-you-users here to the effect of "i agree to be added to this maintainer group"